vignettes/finalsize.Rmd

```diff
@@ -49,128 +48,63 @@ knitr::opts_chunk$set(
+## Getting population estimates
 
-The contact data are divided into five age groups: 0 -- 4, 5 -- 17, 18 -- 39, 40 -- 64, and 65 and over, specified using the `age.limits` argument in `socialmixr::contact_matrix()`.
-The `symmetric = TRUE` argument to `socialmixr::contact_matrix()` returns a symmetric contact matrix, so that the contacts reported by group $\{i\}$ of individuals from group $\{j\}$ are the same as those reported by group $\{j\}$ of group $\{i\}$.
-
-The demographic data --- the number of individuals in each age group --- is also available through `socialmixr::contact_matrix()`.
+Population estimates at the country scale are relatively easy to get from trusted data aggregators such as [Our World in Data](https://ourworldindata.org/world-population-growth).
+More detailed breakdowns of population estimates at the sub-national scale may be available from their respective national governments.
+Here, we use an estimate for the U.K. population size of about 67 million.
 
 ```{r}
-# get UK polymod data
-polymod <- socialmixr::polymod
-contact_data <- socialmixr::contact_matrix(
-  polymod,
-  countries = "United Kingdom",
-  age.limits = c(0, 5, 18, 40, 65),
-  symmetric = TRUE
-)
-
-# view the elements of the contact data list
-# the contact matrix
-contact_data$matrix
-
-# the demography data
-contact_data$demography
-
-# get the contact matrix and demography data
-contact_matrix <- t(contact_data$matrix)
-demography_vector <- contact_data$demography$population
-demography_data <- contact_data$demography
-
-# scale the contact matrix so the largest eigenvalue is 1.0
-# this is to ensure that the overall epidemic dynamics correctly reflect
-# the assumed value of R0
-contact_matrix <- contact_matrix / max(Re(eigen(contact_matrix)$values))
+# get UK population size
+uk_pop <- 67 * 1e6
+```
 
-# divide each row of the contact matrix by the corresponding demography
-# this reflects the assumption that each individual in group {j} make contacts
-# at random with individuals in group {i}
-contact_matrix <- contact_matrix / demography_vector
+This initial example assumes uniform mixing, i.e., that all individuals in the population have a similar number of social contacts. This can be modelled in the form of a single-element contact matrix, which must be divided by the population size.
 
-n_demo_grps <- length(demography_vector)
+```{r}
+# prepare contact matrix
+contact_matrix <- matrix(1.0) / uk_pop
 ```
 
 ::: {.alert .alert-secondary}
+Social contacts are well known to be non-uniform, with age being a strong influence on how many contacts a person has, and moreover, on the ages of their contacts. A relatively simple example is that of children of school-going age, who typically have more social contacts than the elderly, and most of whose social contacts are with other schoolchildren.
```

Comment:
Minor edit from `has, and moreover,` to `has and, moreover,`

```
Social contacts are well known to be non-uniform, with age being a strong influence on how many contacts a person has and, moreover, on the ages of their contacts. A relatively simple example is that of children of school-going age, who typically have more social contacts than the elderly, and most of whose social contacts are with other schoolchildren
```

Reply:
No more comments for the moment. 

In closing, I agree that the uniform-mixing examples simplify access for users with the most minimal version of the package. It is also a way to justify the assumptions behind the necessity to account for heterogeneous susceptibility in the following vignettes. Also, this could also be helpful as a baseline scenario for comparisons.

Reply:
Fixed in https://github.com/epiverse-trace/finalsize/pull/143/commits/2dee45a5fc36375219784f1b66ef62e62eb42ccb